### PR TITLE
Don't delete `.npmrc` in CI

### PR DESCRIPTION
**Pull request by armanbilge**
This caused problems in https://github.com/gemini-hlsw/explore/pull/1502.

I actually don't know why it's like this—I just copied the old CI. But now I'm suspicious, so I removed the code that doesn't make sense. Let's see if it works.

Edit: lol that didn't take long...

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -251,12 +251,7 @@ lazy val sbtStage = WorkflowStep.Sbt(List("stage"), name = Some("Stage"))
 
 // https://stackoverflow.com/a/55610612
 lazy val npmInstall = WorkflowStep.Run(
-  List(
-    "rm .npmrc",
-    """npm config set "@fortawesome:registry" https://npm.fontawesome.com/""",
-    """npm config set "//npm.fontawesome.com/:_authToken" ${{ secrets.FONTAWESOME_NPM_AUTH_TOKEN }}""",
-    "npm install"
-  ),
+  List("npm install"),
```
Comment (cquiroz) [inline build.sbt:254]: Do we need to restore `.npmrc`?

Comment (armanbilge) [reply build.sbt:254]: No, I don't think we should delete it. I think all these steps were a bad hack.

**Comment by armanbilge** [comment]
Ok, I think it's fixed 👍

**Comment by cquiroz** [comment]
Cool, thanks

**Review by cquiroz — APPROVED** [review]

**Pull request merged**